README.md

```diff
@@ -102,6 +102,44 @@ pdk test unit
 
 This command runs all available unit tests.
 
+### Convert an existing module
+
+Aside from creating new modules, the PDK now has the ability to convert an existing module into a PDK ?enabled? module. You can then manage and test your module using the tools bundled with the PDK.
+
+To convert a module you must first decide on a template to convert to. This as default is set as ?PUPPET TEMPLATE?, however you can use your own template by passing it in as a parameter to the command:
+```
+pdk convert ?--template_url? my/template/location (the exact flag tbd)
```

Comment:
`--template-url` is explained at https://puppet.com/docs/pdk/1.0/pdk_reference.html

Reply:
(and this will use the same definition)